src/shogun/regression/svr/LibLinearRegression.cpp

```diff
@@ -93,7 +93,10 @@ bool CLibLinearRegression::train_machine(CFeatures* data)
 	}
 	prob.l=num_vec;
 	prob.x=features;
-	prob.y=SG_MALLOC(float64_t, prob.l);
+    prob.y=SG_MALLOC(float64_t, prob.l);
```

Comment:
there are some style issues, we are using tab indent, see https://travis-ci.org/shogun-toolbox/shogun/jobs/442635134#L728

Reply:
Thanks! I have to switch my IDE to indent!